tests/phpunit/RetryTest.php

```diff
@@ -1,606 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Keboola\DbExtractor\Tests;
-
-use Keboola\CommonExceptions\UserExceptionInterface;
-use Keboola\Csv\CsvWriter;
-use Keboola\DbExtractor\Adapter\PDO\PdoConnection;
-use Keboola\DbExtractor\Adapter\PDO\PdoExportAdapter;
-use Keboola\DbExtractor\Application;
-use Keboola\DbExtractorConfig\Configuration\ValueObject\ExportConfig;
-use Keboola\DbExtractor\Exception\UserException;
-use Keboola\DbExtractor\Extractor\Common;
-use Keboola\DbExtractor\Test\ExtractorTest;
-use Monolog\Logger;
-use Keboola\Temp\Temp;
-use Monolog\Handler\TestHandler;
-use PDO;
-use PHPUnit\Framework\Assert;
-use Symfony\Component\Debug\ErrorHandler;
-
-class RetryTest extends ExtractorTest
```

Comment:
I removed the test `RetryTest`.
It will be replaced in next PR with something better.
This test modified the internal values in objects using reflection, and it is not OK.
In addition, it sometimes accidentally did not pass to CI.

... so, Marc suggested to solve the problems:
https://keboola.slack.com/archives/CQ1ASK06M/p1597306051029200?thread_ts=1597233995.016300&cid=CQ1ASK06M